index.html

```diff
@@ -464,6 +464,12 @@ <h3>Storage Services</h3>
             (Verifier) &bull; Storage Service (Holder)
           </strong>
         </p>
+      <section>
+        <h3>Workflow Service</h3>
+        <p>
+          The Workflow Service provides a way for coordinators to automate
+          specific interactions for specific users. Each role: Holder, Issuer, and Verifier, may run their own Workflow Service to create and manage exchanges that realize particular workflows. Administrators configure the workflow system to support particular flows. Then, when the business rules justify it, coordinators create exchanges at their Workflow Service and give authorized access to those exchanges to any party. 
```

Comment:
Wrapped to match the rest of this source. Tweaked RFC2119 language, as this section is informative. Other changes are not properly highlighted, but do exist.
```suggestion
The Workflow Service provides a way for coordinators to automate specific
interactions for specific users. Each role (Holder, Issuer, and Verifier)
can run their own Workflow Service to create and manage exchanges that
realize particular workflows. Administrators configure the workflow system
to support particular flows. Then, when the business rules justify it,
coordinators create exchanges at their Workflow Service and give authorized
access to those exchanges to any party. 
```